README.md

```diff
@@ -39,16 +39,37 @@ JGR-Space Physics [Publication](https://doi.org/10.1029/2018JA025297)
 
 # Installation
 ## Starting from scratch
-* One simple way to get a complete science python package is from [enthought](https://store.enthought.com)
-* at command line type
+* Python and associated packages for science are freely available. Convenient science python package setups are available from https://www.python.org/ and [Anaconda](https://www.anaconda.com/distribution/). Anaconda also includes a developer environment. Core science packages such as numpy, scipy, matplotlib, pandas and many others may also be installed directly via pip or your favorite package manager.
+
+* Installation through pip
 ```
 pip install pysat
 ```
-* in python, run pysat.utils.set_data_dir('path to top level data dir')
+* Installation through github
+```
+git clone https://github.com/pysat/pysat.git
+cd pysat
+python setup.py install
+```
+An advantage to installing through github is access to the development branches.  The latest bugfixes can be found in the `develop` branch.
```

Comment:
```suggestion
An advantage to installing through github is access to the development branches.  The latest bugfixes can be found in the `develop` branch.  However, this branch is not stable (as the name implies).
```

Reply:
It may also be good to suggest using this branch only in a virtual environment or with the `python setup.py develop`.